.env.example

```diff
@@ -1,4 +1,4 @@
 DB_HOST=localhost
 DB_USER=myuser
 DB_PASS=mypassword
```

Comment:
we don't need db credentials here right?

Reply:
Yes. We don't need db credentials here